tests/travis/install-extension-manager.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+set -ex
+
+BASE_PATH=$(pwd)
+MW_INSTALL_PATH=$BASE_PATH/../mw
+
+# Run Composer installation from the MW root directory
+function installToMediaWikiRoot {
+	echo -e "Running MW root composer install build on $TRAVIS_BRANCH \n"
+
+	cd $MW_INSTALL_PATH
+
+	if [ "$PHPUNIT" != "" ]
+	then
+		composer require 'phpunit/phpunit='$PHPUNIT --prefer-source --update-with-dependencies
+	else
+		composer require 'phpunit/phpunit=3.7.*' --prefer-source --update-with-dependencies
+	fi
+
+	if [ "$VERSION" != "" ]
```

Comment:
Not sure if I run into trouble on Travis with `$VERSION` because it was already occupied.